### PR TITLE
Remove Coturn brokering

### PR DESCRIPTION
**Pull request by Brutus5000**
This featured has been moved to a dedicated API.

### CODE REVIEW

File: integration_tests/test_server.py [hunk 1]
```diff
@@ -6,20 +6,6 @@ async def test_ping(client_factory):
...
-
-    await client.send_command("ice_servers")
-    msg = await client.read_until_command("ice_servers")
-
-    assert msg["ttl"] > 60
-    assert msg["ice_servers"]
-    for server in msg["ice_servers"]:
-        assert server["urls"]
-        assert server["username"]
-        assert server["credential"]
```
Comment (Askaholic) [inline integration_tests/test_server.py:20]: I think we probably want to keep this actually, but adding a `# DEPRECTATED:` comment would be good so it doesn't get forgotten when the command is removed.

**Review by Askaholic — COMMENTED** [review]
I don't think the Pipfile.lock actually needs to be updated. I think that happened because of the changes you made when you removed twilio. I can fix it up for you though.

File: integration_tests/test_server.py (hunk 1 above)
Comment (Brutus5000) [reply integration_tests/test_server.py:20]: There still is `test_ice_servers_empty`. Isn't that sufficient? (since it will always be empty)

Comment (Askaholic) [reply integration_tests/test_server.py:20]: Ah, in that case, that's fine

**Review by Askaholic — APPROVED** [review]

**Pull request merged**